modules/gapi/include/opencv2/gapi/cpu/stereo.hpp

```diff
@@ -0,0 +1,25 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#ifndef OPENCV_GAPI_CPU_STEREO_API_HPP
+#define OPENCV_GAPI_CPU_STEREO_API_HPP
+
+#include <opencv2/gapi/gkernel.hpp> // GKernelPackage
+
+namespace cv {
+namespace gapi {
+namespace ster {
```

Comment:
Why `ster`? 
Where did the `stereo` go?

Reply:
In this case there is name collision of `namespace stereo` and operation `stereo`

Reply:
Please discuss namespace name with the team -- to comply with OpenCV modules names we should create `calib3d` namespace, but we are focusing on addition of stereoscopic functionality, so may be namespace should be named as `stereo` or `stereoscopic`.

Reply:
`cv::gapi::ster` -> `cv::gapi::calib3d`